Sascha-Gschwind/incubator-superset
# Added menu link for quick upload a csv

### CATEGORY

Choose one

- [ X ] Enhancement (new features, refinement)

### SUMMARY
Added a new menu link for the feature "Quick upload a CSV".

### TEST PLAN
1. Update the database
2. Open Superset
3. Choose "Sources"
Do you see the link "Quick upload a CSV"?

### ADDITIONAL INFORMATION


- [ X ] Changes UI
- [ X ] Introduces new feature or API

### REVIEWERS
BA-Team
